Extend/Model/Payment.php

```diff
@@ -56,6 +57,14 @@ public function getLogoUrl(): ?string
             return null;
         }
 
+        if ($this->oxpayments__oxid->value === SofortPaymentMethod::getName(true)) {
+            return sprintf(
```

Comment:
Aren't we missing the base path here?

Reply:
The full absolute URL is stored in the `wdoxidee_logo` field